tests/backends/storage/test_swift.py

```diff
@@ -11,10 +11,10 @@
 from ralph.exceptions import BackendException, BackendParameterException
 
 
-def test_swift_storage_instantiation_failure_should_raise_exception(
+def test_backends_database_storage_swift_storage_instantiation_failure_should_raise_exception(
```

Comment:
I think you can `sed` this recursively: `"s/test_backends_database_storage/test_backends_storage/g"` :sweat_smile: